.github/workflows/tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     name: static-checks (go ${{ matrix.go_version }})
     needs: resolve-versions
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 20
```

Comment:
Another GH workflow related change, it seems that `setup-go` is taking an extra 2 minutes to setup Go `1.22.x`, not leaving enough time for the rest of the job to run 🙂 : https://github.com/hashicorp/terraform-exec/actions/runs/8914407327/job/24482771576?pr=447